AirCasting/SessionViews/ChartViewModel.swift

```diff
@@ -20,7 +20,7 @@ final class ChartViewModel: ObservableObject {
         }
     }
     @Injected private var persistence: PersistenceController
-    @Injected private var settings: UserSettings
+    @InjectedObject var settings: UserSettings
```

Comment:
this needs to stay private, the whole purpose of `ViewModels` is to hide that kind of objects behind a UI-domain interface